ang_scripts/force_password_change.py

```diff
@@ -0,0 +1,17 @@
+import django
+import logging
+
+logger = logging.getLogger(__name__)
+
+django.setup()
+
+
+def force_password_change():
+    from registry.groups.models import CustomUser
```

Comment:
Imports at top of the file, please.

Reply:
Doesn't work in this case as this import needs to be done after django.setup() executes